routers/repo/issue.go

```diff
@@ -1055,6 +1055,34 @@ func UpdateIssueTitle(ctx *context.Context) {
 	})
 }
 
+// UpdateIssueRef change issue's ref (branch)
+func UpdateIssueRef(ctx *context.Context) {
+	issue := GetActionIssue(ctx)
+	if ctx.Written() {
+		return
+	}
+
+	if !ctx.IsSigned || (!issue.IsPoster(ctx.User.ID) && !ctx.Repo.CanWriteIssuesOrPulls(issue.IsPull)) {
+		ctx.Error(403)
+		return
+	}
+
+	ref := ctx.QueryTrim("ref")
+	if len(ref) == 0 {
+		ctx.Error(204)
```

Comment:
How can the user remove the target branch?

Reply:
So far it can't, however, that's the same problem as when you try to create an issue - once selected, the branch can't be removed short of refreshing the page. I'll look into it